Sources/Common/Transform/LandmarkTransform/index.js

```diff
@@ -13,23 +13,6 @@ function vtkLandmarkTransform(publicAPI, model) {
   // Set our className
   model.classHierarchy.push('vtkLandmarkTransform');
 
-  // Convert a mat4 matrix to a Matrix 2 dimensions
-  function mat4To2DArray(mat) {
```

Comment:
it seems that you are actually doing such conversion in some parts of your code, don't you ? 
If so, it means there is a need and it would reduce code duplciation.

Reply:
Actually I think it is espacially the rowsToMat3 that is usefull. I created all of them so if in further refactorings they can be used.